cyclops-ctrl/internal/models/dto/k8s.go

```diff
@@ -248,6 +248,48 @@ func (s *StatefulSet) SetDeleted(deleted bool) {
 	s.Deleted = deleted
 }
 
+type PersistentVolumeClaim struct {
+	Group       string 							`json:"group"`
+	Version     string 							`json:"version"`
+	Kind        string 							`json:"kind"`
+	Name        string 							`json:"name"`
+	Namespace   string 							`json:"namespace"`
+	AccessModes []v1.PersistentVolumeAccessMode `json:"accessmodes"`
+	Size        string							`json:"size"`
```

Comment:
You will need to add
```go
	Deleted   bool   `json:"deleted"`
```
in order for GetDeleted to work

Reply:
Added!